src/components/tab/EmptyContent.vue

```diff
@@ -4,18 +4,49 @@
 			<img :src="flowSvg" alt="flow">
 		</div>
 		<div class="title text-center">
-			{{ t('integration_openproject', "No OpenProject links yet") }}
+			{{ emptyContentMessage }}
+		</div>
+		<br>
+		<div v-if="state === 'no-token' || state === 'error'" class="connect-button">
```

Comment:
```suggestion
		<div v-if="['error', 'no-token].includes(state)" class="connect-button">
```
you can also use this logic from the `computed` section, just to make HTML clean
```
<div v-if="someComputedMethod" class="connect-button">
```

Reply:
done